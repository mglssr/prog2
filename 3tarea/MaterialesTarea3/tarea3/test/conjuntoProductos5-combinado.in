crearConjuntoProductos 10
cardinalConjuntoProductos
cantMaxConjuntoProductos
insertarConjuntoProductos 0
insertarConjuntoProductos 1
insertarConjuntoProductos 2
imprimirConjuntoProductos
borrarDeConjuntoProductos 1
insertarConjuntoProductos 3
insertarConjuntoProductos 6
insertarConjuntoProductos 7
borrarDeConjuntoProductos 2
imprimirConjuntoProductos
insertarConjuntoProductos 8
insertarConjuntoProductos 9
insertarConjuntoProductos 1
insertarConjuntoProductos 2
cardinalConjuntoProductos
cantMaxConjuntoProductos
imprimirConjuntoProductos
unionConjuntoProductos 5 1 3 4 5 6
interseccionConjuntoProductos 5 1 3 4 5 6
diferenciaConjuntoProductos 5 1 3 4 5 6
borrarDeConjuntoProductos 1
borrarDeConjuntoProductos 2
unionConjuntoProductos 5 1 3 4 5 6
interseccionConjuntoProductos 5 1 3 4 5 6
diferenciaConjuntoProductos 5 1 3 4 5 6
borrarDeConjuntoProductos 8
borrarDeConjuntoProductos 9
unionConjuntoProductos 5 1 3 4 5 6
interseccionConjuntoProductos 5 1 3 4 5 6
diferenciaConjuntoProductos 5 1 3 4 5 6
liberarConjuntoProductos
Fin
