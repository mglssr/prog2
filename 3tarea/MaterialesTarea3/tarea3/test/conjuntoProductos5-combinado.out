Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> El conjunto productos fue creado en forma correcta.
2> El cardinal del conjunto es: 0.
3> La cantidad maxima del conjunto es: 10.
4> Se ejecuto 'insertarTConjuntoProductos' con producto de id 0 exitosamente.
5> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
6> Se ejecuto 'insertarTConjuntoProductos' con producto de id 2 exitosamente.
7> Conjunto de productos:
0 1 2 
8> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 1 exitosamente.
9> Se ejecuto 'insertarTConjuntoProductos' con producto de id 3 exitosamente.
10> Se ejecuto 'insertarTConjuntoProductos' con producto de id 6 exitosamente.
11> Se ejecuto 'insertarTConjuntoProductos' con producto de id 7 exitosamente.
12> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 2 exitosamente.
13> Conjunto de productos:
0 3 6 7 
14> Se ejecuto 'insertarTConjuntoProductos' con producto de id 8 exitosamente.
15> Se ejecuto 'insertarTConjuntoProductos' con producto de id 9 exitosamente.
16> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
17> Se ejecuto 'insertarTConjuntoProductos' con producto de id 2 exitosamente.
18> El cardinal del conjunto es: 8.
19> La cantidad maxima del conjunto es: 10.
20> Conjunto de productos:
0 1 2 3 6 7 8 9 
21> Conjunto de productos union:
0 1 2 3 4 5 6 7 8 9 
22> Conjunto de productos interseccion:
1 3 6 
23> Conjunto de productos diferencia:
0 2 7 8 9 
24> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 1 exitosamente.
25> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 2 exitosamente.
26> Conjunto de productos union:
0 1 3 4 5 6 7 8 9 
27> Conjunto de productos interseccion:
3 6 
28> Conjunto de productos diferencia:
0 7 8 9 
29> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 8 exitosamente.
30> Se ejecuto 'borrarDeTConjuntoProductos' con producto de id 9 exitosamente.
31> Conjunto de productos union:
0 1 3 4 5 6 7 
32> Conjunto de productos interseccion:
3 6 
33> Conjunto de productos diferencia:
0 7 
34> Conjunto de productos liberado exitosamente.
35> Fin.
